media/src/Settings.svelte

```diff
@@ -115,72 +149,126 @@
-               step=".02"
-               bind:value={scale}
-               on:change={rescale}
-               /><span class="output">{scala}</span></span>
+    <label class="form-label" for="scale">Scale</label>
+    <span class="form-range">
+        <input
+            type="range"
+            name="scale"
+            id="scale"
+            min="-2"
+            max="3"
+            step=".02"
+            bind:value={scale}
+            on:change={rescale}
+        >
+        <span class="output box-23 text-right">{scala}</span>
+    </span>
 
-    <span class="box-1 text-right">grid</span>
-    <label class="switch box-2">
+    <div class="form-check form-switch">
+        <label class="form-check-label box-1" for="gridVisible">Grid</label>
         <input
+            class="form-check-input box-1"
             type="checkbox"
             name="gridVisible"
             id="gridVisible"
+            role="switch"
+            aria-checked="true"
             bind:checked={gridMeshes.visible}
             on:change={render}
-            />
-        <span class="slider round" />
-    </label>
+        />
+    </div>
 
-    <span class="box-1 text-right">ortho/proj</span>
-    <label class="switch box-2">
+    <div class="form-check form-switch">
+        <label class="form-check-label box-2" for="orthoCamera">Linear Perspective</label>
         <input
+            class="form-check-input box-2"
             type="checkbox"
+            role="switch"
+            aria-checked="false"
             name="orthoCamera"
             id="orthoCamera"
             bind:checked={orthoCamera}
             on:change={render}
-            />
-        <span class="slider round" />
-    </label>
+        />
+    </div>
+</div>
+
+<div
+    class="settings-box"
+    class:grid={showUpload}
+    hidden={!showUpload}
+    id="upload-box"
+>
+    <button
+        type="button"
+        class="btn btn-sm btn-light box-3"
+        style="margin-left: auto;"
+        on:click={() => {
+            showUpload = false;
+        }}
+    >
+        <i class="bi bi-x-lg"></i>
+    </button>
+    <label for="upload">Upload a scene</label>
+    <input id="upload" type="file" accept="application/json"
+        on:change={(e) => {uploadScene(e);}}
+    >
 </div>
 
 <button
     class="button"
     id="settings"
     title="Settings"
-    on:click={() => {
-    showSettings = !showSettings;
-    }}><i class="fa fa-cog" /></button>
+    on:click={() => { 
+        showUpload = false;
+        showSettings = !showSettings; 
+    }}
+>
+    <i class="fa fa-cog" />
+</button>
 <button class="button" id="encodeURL" title="Encode URL" on:click={encode}>
     <i class="fa fa-barcode" />
 </button>
+<button 
+    class="button" 
+    id="load-scene" 
+    title="Upload Scene" 
+    on:click={() => { 
+        showSettings = false;
+        showUpload = !showUpload;
+    }}
+>
+    <i class="fa fa-upload" />
+</button>
+<button 
+    class="button"
+    id="download-scene"
```

Comment:
These button id's aren't being used, so I would just remove then.

Reply:
I only deleted the ids that I made, but there might be more that were there from Drew's prototype. Would you like me to delete those as well?